### PR TITLE
bump k8s libs

### PR DESCRIPTION
**Pull request by xh3b4sd**
Towards https://github.com/giantswarm/giantswarm/issues/2844.

### CODE REVIEW

**Review by rossf7 — APPROVED** [review]
LGTM after the libs are merged.

**Comment by xh3b4sd** [comment]
Successfully tested on geckon.

**Review by kopiczko — APPROVED** [review]

**Review by fgimenez — APPROVED** [review]
LGTM

**Pull request merged**
